inc/classes/admin/settings/class-page.php

```diff
@@ -1150,6 +1161,16 @@ private function preload_section() {
 					'default'           => [],
 					'sanitize_callback' => 'sanitize_textarea',
 				],
+				'preload_fonts' => [
+					'type'              => 'textarea',
+					'label'             => __( 'Fonts to preload', 'rocket' ),
+					'description'       => __( 'Specify urls of the font files to be preloaded (one per line)', 'rocket' ),
```

Comment:
Can you add an `helper` entry to also display what was suggested here: https://github.com/wp-media/wp-rocket/issues/2543#issuecomment-615234025

Reply:
@Tabrisrp Done in https://github.com/wp-media/wp-rocket/pull/2554/commits/cd4ead9016d28ce7e6a7cfc04ca864c212672a3f.